translations/README-uz.md

```diff
@@ -1257,46 +1257,47 @@ Computer science da graflar bir talay masalalarni tasvirlashda ishlatiladi. Shun
 
 ---
 
-## Coding Question Practice
+## Coding bo'yicha savollar ustida ishlash
 
-Now that you know all the computer science topics above, it's time to practice answering coding problems.
+Yuqoridagi barcha "computer science" mavzularini bilb oldingiz, endi coding bo'yicha masalalarga javob berishni mashq qilish vaqti keldi.
 
-**Coding question practice is not about memorizing answers to programming problems.**
+**Codingda masalalarini yechishga tayyorlanish bu javoblarni yodlab qolish emas **
 
-Why you need to practice doing programming problems:
-- Problem recognition, and where the right data structures and algorithms fit in
-- Gathering requirements for the problem
-- Talking your way through the problem like you will in the interview
-- Coding on a whiteboard or paper, not a computer
-- Coming up with time and space complexity for your solutions
-- Testing your solutions
+Masalalar yechishni mashq qilish nima uchun zarur:
+- Muammoni aniqlash va kerakli ma'lumotlar tuzilmalari va algoritmlari mos keladigan joyni topa olish
+- Masala uchun kerakli talablarni to'plash
+- Xuddi intervyudagi kabi o'zingizni usulingizni muammoni hal qilishda ko'rsatish
+- Doskada yoki qog'ozda kod yozish, kompyuterda emas
+- Ish joyi xarajati va ish vaqti xarajatini inobatga olgan holda masalaga yechim topish
```

Comment:
Bu yerda time and space complexityni inobatga olgan holda yechish emas, yechimlarizga time and space complexity qancha bulishini topish deyapti. Suggestion: "Yechimlarizga ish joyi xarajati va ish vaqti xarajatini aniqlash"